src/pages/FishListPage.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import FishCard from '../components/FishCard.js';
+import Header from '../components/Header.js';
+
+export default function FishList({ fishes }) {
+  return (
+    <PageContainer>
+      <Header />
+      <FishlistContainer>
+        {fishes && fishes.map(fish => <FishCard key={fish.Id} fish={fish} />)}
```

Comment:
To make it more complete, you could write a message/error which is shown to the user, when there is no fish data.